tests/test_socket_server.py

```diff
@@ -64,7 +64,7 @@ async def read(self):
         data : `dict`
             A dictionary with objects representing the string read.
         """
-        read_bytes = await asyncio.wait_for(self.reader.readuntil(b"\r\n"), timeout=1)
+        read_bytes = await asyncio.wait_for(self.reader.readuntil(b"\r\n"), timeout=5)
```

Comment:
Please consider using a named constant for the timeout. Then you can change it in one place.

Reply:
Done